app/models/concerns/activity_message_handler.rb

```diff
@@ -5,6 +5,7 @@ module ActivityMessageHandler
   include LabelActivityMessageHandler
   include SlaActivityMessageHandler
   include TeamActivityMessageHandler
+  include LinearActivityMessageHandler
```

Comment:
lets do the same change which we discussed for CSAT service PR.  we can decouple the activity generation to a separate service class. no need to have these included in the conversation module.